yarn.lock

```diff
@@ -1251,6 +1251,98 @@
+    "@fluentui-react-native/immutable-merge" "^1.1.3"
+    "@fluentui-react-native/memo-cache" "^1.1.3"
+    "@fluentui-react-native/merge-props" ">=0.3.3 <1.0.0"
+    "@fluentui-react-native/theme-types" ">=0.11.2 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.9.20 <1.0.0"
+    "@fluentui-react-native/use-slot" ">=0.1.5 <1.0.0"
+    "@fluentui-react-native/use-slots" ">=0.5.7 <1.0.0"
+    "@fluentui-react-native/use-styling" ">=0.6.7 <1.0.0"
+    "@fluentui-react-native/use-tokens" ">=0.1.5 <1.0.0"
+
+"@fluentui-react-native/framework@0.5.33":
+  version "0.5.33"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/framework/-/framework-0.5.33.tgz#ef3474dec99d0d36d5beb15efb4b4e26085e582d"
+  integrity sha512-7FKLbDzfyWCA7pfiUQ+XOzc9iho8NHUtkLFfZMzRGJjf4DYy5Fv7hNez5Zf4mmcl4JQptrOXowXMRcyRRWtOqQ==
+  dependencies:
+    "@fluentui-react-native/composition" ">=0.6.8 <1.0.0"
+    "@fluentui-react-native/default-theme" ">=0.7.13 <1.0.0"
+    "@fluentui-react-native/immutable-merge" "^1.1.3"
+    "@fluentui-react-native/memo-cache" "^1.1.3"
+    "@fluentui-react-native/merge-props" ">=0.3.3 <1.0.0"
+    "@fluentui-react-native/theme-types" ">=0.11.3 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.9.21 <1.0.0"
+    "@fluentui-react-native/use-slot" ">=0.1.5 <1.0.0"
+    "@fluentui-react-native/use-slots" ">=0.5.7 <1.0.0"
+    "@fluentui-react-native/use-styling" ">=0.6.7 <1.0.0"
+    "@fluentui-react-native/use-tokens" ">=0.1.5 <1.0.0"
+
+"@fluentui-react-native/icon@0.8.14":
+  version "0.8.14"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/icon/-/icon-0.8.14.tgz#6f6ff09194148daf0e672158759b626ca5295b4b"
+  integrity sha512-MpmR624+wJU6w/D9lxKsbAZLjU5N9Bd3mn+vT9gQfXSAKaJJUHNFiGMCwb/umzePI9pplRDGXLx26Yrsrt/oww==
+  dependencies:
+    "@fluentui-react-native/default-theme" ">=0.7.12 <1.0.0"
+    "@fluentui-react-native/framework" "0.5.32"
+    "@fluentui-react-native/text" "^0.10.11"
+    "@fluentui-react-native/theme-types" ">=0.11.2 <1.0.0"
+    "@fluentui-react-native/theming-utils" ">=0.5.2 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.9.20 <1.0.0"
+    "@uifabricshared/foundation-compose" "^1.10.9"
+    react-native-svg "^12.1.1"
+
+"@fluentui-react-native/icon@0.8.16":
+  version "0.8.16"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/icon/-/icon-0.8.16.tgz#c67a632e509aec994d6bd056fb766de84b1a7581"
+  integrity sha512-Q7JsH7hqhAR9Lh+MbboC7Dy6WRlzVJOPjGmOsbem2L0nUNzF4Vq3N2/YRZ0yD/24kS83M1XHQYjz+eCSQg4CDg==
+  dependencies:
+    "@fluentui-react-native/default-theme" ">=0.7.13 <1.0.0"
+    "@fluentui-react-native/framework" "0.5.33"
+    "@fluentui-react-native/text" "^0.10.12"
+    "@fluentui-react-native/theme-types" ">=0.11.3 <1.0.0"
+    "@fluentui-react-native/theming-utils" ">=0.5.3 <1.0.0"
+    "@fluentui-react-native/tokens" ">=0.9.21 <1.0.0"
+    "@uifabricshared/foundation-compose" "^1.10.10"
+    react-native-svg "^12.1.1"
+
```

Comment:
I think these are showing up in the lock because we're using an older version of each of these packages somewhere in one of the package.jsons? "fluentui-react-native/icon" for example is at 0.8.17 in the npm registry
https://www.npmjs.com/package/@fluentui-react-native/icon

Reply:
The versions have been updated and yarn.lock is no longer being modified.